--- a/catalog/clients/python/src/catalog_openapi/models/catalog_model.py
+++ b/catalog/clients/python/src/catalog_openapi/models/catalog_model.py
@@ -85,8 +85,6 @@
         * OpenAPI `readOnly` fields are excluded.
         """
         excluded_fields: Set[str] = set([
-            "create_time_since_epoch",
-            "last_update_time_since_epoch",
             "artifact_counts",
         ])
 
